layouts/partials/collapsible-menu.html

```diff
@@ -21,38 +21,62 @@
 {{ end }}
 {{$currentSection := .Scratch.Get "currentSection" }}
 
+<!--- Sidebar layout --->
+{{ .Scratch.Set "sidebarLayout" "default" }}
+{{ if isset .Site.Params "sidebar_layout" }}
+  {{ .Scratch.Set "sidebarLayout" .Site.Params.sidebar_layout }}
+{{ end }}
+{{ if isset .Page.Params "sidebar_layout" }}
+  {{ .Scratch.Set "sidebarLayout" .Page.Params.sidebar_layout }}
+{{ end }}
+
+{{ $sidebarLayout := .Scratch.Get "sidebarLayout" }}
+
+{{ $classname := "collapsible-menu" }}
+{{ if eq $sidebarLayout "sidebar_block" }}
+  {{ $classname = "sidebar-block" }}
+{{ end }}
+
 <!----------------------- Collapisible start ------------------------------------>
   {{ range .Site.Menus.sidebar }}
     {{$parentItem := . }}
     {{ if and (.HasChildren) (eq $currentSection $parentItem.Identifier) }}
-    <div class="collapsible-menu container hidden-lg hidden-md visible-sm visible-xs">
-      <div class="container collapsible-menu-item related-content">
+    <div class="{{ $classname }} margin-top-10 container hidden-lg hidden-md visible-sm visible-xs">
```

Comment:
If this is to fix the lack of space for menus with children, this should be margin bottom not top. Margin top will cause some pages to have a pushed down menu as the first entry. While not huge, its a consistency and attention to detail we should pay attention to.

Reply:
Nope, this is for fixing big spacing when using Eric's style on mobile view. He added a `margin-top: 30px` for the whole sidebar (for desktop). In that case, my sidebar on mobile will have 30px spacing between the breadcrumb, and I feel it too much.

![image](https://user-images.githubusercontent.com/39588094/109700734-299a7f80-7b60-11eb-82f1-67b66440d6fa.png)

Reply:
I personally think the 30px gives the menu some breathing room and looks good. Negative space is a useful design practice for making a UI feel less busy/compressed @chrisguindon thoughts?

Reply:
How about we meet in the middle with 20px?

Reply:
Updated to 20px.